test/benchmark/suite/render.go

```diff
@@ -234,7 +237,18 @@ func renderMetricsTable(writer io.Writer, headerSettings []ReportTableHeader, re
 	return nil
 }
 
-func renderMetricsTableHeader(table *tabwriter.Writer, headers []ReportTableHeader) {
+func renderProfilesTable(writer io.Writer, target, key string, titleLevel int, reports []*BenchmarkReport) {
```

Comment:
blocked by https://github.com/envoyproxy/gateway/pull/3709, need to update the doc later